### DISCUSSION TITLE
Storage devices not found - Nwipe should be run as root or sudo

### DISCUSSION THREAD

**Discussion by MoritzB91** [PartialVolume/shredos.x86_64#239] [General]
Dear Community,

I would like to erase the SSDs of two Dell laptops (1x [Dell XPS 13 Plus 9320 (2022) Platinum, Core i7-1260P, 32GB RAM, 1TB SSD + [Dell XPS 15 9510 (2021) Touch Platinum Silver, Core i7-11800H, 16GB RAM, 1TB SSD, GeForce RTX 3050 Ti) using the latest build of ShredOS for USB drives  (v2023.08.2_25.0_x86-64_0.35), unfortunately everytime the following error message appears (see screenshot for full details attached):
```
info: Storage devices not found. Nwipe should be run as root or sudo/su, i.e sudo nwipe etc.
``` 
Any idea what needs to be done to get Nwipe to run properly here?

Thanks in advance!
![20240404_160225](https://github.com/PartialVolume/shredos.x86_64/assets/166027983/6bcab0a4-4a2a-4b3c-a54b-1978aa68416a)

**Comment by PartialVolume** [comment]
Have you checked that the storage menu in the bios is set for AHCI?

**Reply by MoritzB91** [reply]
Hi there,

thanks for your help on this. For the Dell XPS 13 Plus 9320 (2022) switching to AHCI worked well. For the Dell XPS 15 9510 (2021) with GeForce RTX 3050 Ti the graphic card seems to be the problem. Right now I always receive the following error message after ShredOS was loaded 100% => 
```
Unable to create qla2xxx trace instance, instace logging will be disabled.
``` 

I added **nomodeset** or **nomodeset noapic** or **noapic** as adviced in another post to both /boot/grub.cfg and /EFI/BOOT/grub.cfg but this doesn't do the trick yet. It even makes it worse, resulting in a black screen after loading. On the laptop the only thing connected is a USB-C HUB with the USB-Stick with ShredOS on it, no monitor, nothing else. The HUB contains an HDMI port, might this be the reason in regards to the DRM topic?

**Reply by Firminator** [reply]
That's a Qlogic HBA driver. PartialVolume removed a bunch of them in the past, but it seems this one he missed. You can ignore the error since you don't have a FiberChannel card supported by this driver.

**Comment by PartialVolume** [comment]
Yes, it could be the HDMI port, In the bios Dell may have an option for selecting the default display adapter as HDMI, RTX 3050 or auto, so switching to RTX 3050 may do the trick. I've had that problem with other Dell hardware that was fixed by selecting the appropriate display hardware in the bios.
